frontends/mit-open/src/pages/HomePage/PersonalizeSection.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react"
+import { Typography, styled, Container, ButtonLink } from "ol-components"
+import { useUserMe } from "api/hooks/user"
+import * as urls from "@/common/urls"
+
+const FullWidthBackground = styled.div(({ theme }) => ({
+  padding: "80px 0",
+  background:
+    'url("/static/images/homepage/personalize-bg.png") center top no-repeat',
+  backgroundSize: "cover",
+  [theme.breakpoints.down("md")]: {
+    padding: "40px 0",
+  },
+}))
+
+const PersonalizeContainer = styled(Container)(({ theme }) => ({
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  gap: "80px",
+  [theme.breakpoints.down("sm")]: {
+    gap: "28px",
+    flexDirection: "column",
+  },
+}))
+
+const ImageContainer = styled.img(({ theme }) => ({
+  display: "flex",
+  flex: 1,
+  alignItems: "end",
+  minWidth: "0px",
+  [theme.breakpoints.down("sm")]: {
+    maxWidth: "100%",
+  },
+}))
+
+const ControlsContainer = styled.section(({ theme }) => ({
+  maxWidth: "442px",
+  flex: 1,
+  color: theme.custom.colors.white,
+  display: "flex",
+  alignItems: "start",
+  flexDirection: "column",
+  gap: "32px",
+}))
+const TextContainer = styled.div({
+  display: "flex",
+  flexDirection: "column",
+  gap: "20px",
+})
+
+const PersonalizeContent: React.FC = () => {
+  const { data: user, isLoading } = useUserMe()
+
+  if (isLoading) {
+    return null
```

Comment:
Can we add a placeholder for the `ControlsContainer` or put a max width and position the image while loading? While returning `null` here the image takes up the full width causing a brief content jump

<img width="250" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/d33f93a0-8bf7-4d7c-8ba1-2f9e4187c967">

Reply:
done